src/Resolver.hack

```diff
@@ -55,7 +58,26 @@ final class Resolver {
         return $ret;
     }
 
-    public async function resolveImpl(
+    private function parseRequest(string $input): \Graphpinator\Parser\ParsedRequest {
+        $source = new \Graphpinator\Source\StringSource($input);
+        $parser = new \Graphpinator\Parser\Parser($source);
+        try {
+            return $parser->parse();
+        } catch (\Graphpinator\Exception\GraphpinatorBase $e) {
```

Comment:
Would it be hard to update the tokenizer+parser to throw UserFacingErrors so we don't need to transform them here?

Reply:
Yeah, I thought about doing this. It's certainly non-trivial and the payoff didn't seem large, so I punted for now, but I do think we should do it eventually. How about I file an issue?

Reply:
Filed: https://github.com/mwildehahn/hack-graphql/issues/77